region/wreckedship/main.json

```diff
@@ -726,6 +747,94 @@
                       "requires": [ "h_canBombThings" ]
                     }
                   ]
+                },
+                {
+                  "name": "SpeedKeep Bounce (Phantoon Dead)",
+                  "notable": false,
+                  "requires": ["Morph"],
+                  "obstacles": [
+                    {
+                      "id": "C",
+                      "requires": [
+                        "f_DefeatedPhantoon",
+                        "canXRayTurnaround",
+                        "canSpringBallBounce",
+                        {"or": [
+                          "HiJump",
+                          "canSpeedKeepChain"
+                        ]},
+                        {"canShineCharge": {
+                          "usedTiles": 20,
+                          "openEnd": 1,
+                          "shinesparkFrames": 0,
+                          "steepDownTiles": 4
+                        }}
+                      ]
+                    }
+                  ],
+                  "note": [
+                    "Run towards the conveyor to more easily build the ShineCharge, and for an easier bounce into the morph tunnel.",
+                    "Neutral bounce into the Morph tunnel, and SpringBall bounce to continue all the way through."
+                  ],
+                  "devNote": "16 tiles plus conveyor pushback."
+                },
+                {
+                  "name": "Chain SpeedKeep Bounce (Phantoon Alive)",
+                  "notable": false,
+                  "requires": ["Morph"],
+                  "obstacles": [
+                    {
+                      "id": "C",
+                      "requires": [
+                        "canSpeedKeepChain",
+                        "canXRayTurnaround",
+                        "canSpringBallBounce",
+                        {"canShineCharge": {
+                          "usedTiles": 16,
+                          "openEnd": 1,
+                          "shinesparkFrames": 0,
+                          "steepDownTiles": 4
+                        }}
+                      ]
+                    }
+                  ],
+                  "note": [
+                    "Run towards the conveyor for an easier bounce into the morph tunnel.",
+                    "Neutral bounce into the Morph tunnel, and SpringBall bounce to continue all the way through."
+                  ],
+                  "devNote": "Defeating Phantoon makes this strat easier."
+                },
+                {
+                  "name": "SpeedKeep Bounce (Phantoon Alive)",
+                  "notable": false,
+                  "requires": [
+                    {"not": "f_DefeatedPhantoon"},
+                    "canRiskPermanentLossOfAccess",
```

Comment:
```suggestion
```
Can these be removed? Isn't it strictly harder if Phantoon is alive - could this exact strat be done if he's dead, i.e. by not using the full conveyor belt?

Reply:
These two are not possible if the conveyer is on.  Feels like I need about 3 more tiles to get a spark.